src/dictionary.js

```diff
@@ -0,0 +1,24 @@
+const https = require("https");
+const fs = require("fs");
+const generateList = async () => new Promise((resolve, reject) => {
+    let wordList = [];
+    if (fs.existsSync("../dictionary.json")) {
+        wordList = fs.readFileSync("../dictionary.json");
+        resolve(JSON.parse(wordList));
+    } else {
+        https.get("https://raw.githubusercontent.com/dwyl/english-words/master/words_alpha.txt", (res) => {
+            res.on("data", (d) => {
+                wordList = [ ...wordList, ...d.toString().split(/\r?\n/)];
+            });
+            res.on("end", () => {
+                fs.writeFile("../dictionary.json", JSON.stringify(wordList), (err) => {
```

Comment:
If we use a config file, we can avoid hardcoding "dictionary.json"

Reply:
I agree and spun up issue: https://github.com/prosif/homegames/issues/17
to track that